modules/azure/nsg.go

```diff
@@ -0,0 +1,194 @@
+package azure
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-09-01/network"
+)
+
+// GetDefaultNsgRulesClientE returns a rules client
```

Comment:
NIT: expand comment to explain what a rules client is and, more importantly, when/why you'd use it.

Reply:
Added comments to expand on use case.